src/operator/slice_channel-inl.h

```diff
@@ -32,13 +32,13 @@ struct SliceChannelParam : public dmlc::Parameter<SliceChannelParam> {
   bool squeeze_axis;
   DMLC_DECLARE_PARAMETER(SliceChannelParam) {
     DMLC_DECLARE_FIELD(num_outputs).set_lower_bound(1)
-    .describe("Number of outputs to be sliced.");
+    .describe("Number of splits. Note that this should evenly divide the length of the `axis`.");
     DMLC_DECLARE_FIELD(axis).set_default(1)
-    .describe("Dimension along which to slice.");
+    .describe("Axis along which to split.");
     DMLC_DECLARE_FIELD(squeeze_axis).set_default(0)
-    .describe("If true, the dimension will be squeezed."
-              " Also, input.shape[axis] must be the same as `num_outputs`"
-              " when squeeze_axis is turned on.");
+    .describe("If true, removes the dimension `axis` from the output shape"
```

Comment:
This is not exactly as the numpy.squeeze. I changed it to "removes the dimension of length 1 from the output shape."